charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -377,7 +378,13 @@ node:
   # Enable the linux daemonset creation
   enableLinux: true
   enableWindows: false
-  # The "maximum number of attachable volumes" per node
+  # Number of volume attachments reserved for system use.
+  # Not used when --volume-attach-limit is specified.
+  # The total amount of volume attachments for a node is computed as:
```

Comment:
Remove this comment and all other comment lines below it, incorrect on non-nitro instances and not necessary

Reply:
Would it be helpful to point to cmd/options/node_options.go for a better explanation?

Reply:
No, docs should never point to code. Users do not need to know how the math is done internally, which may change from version to version and instance to instance, only what the parameter does.

Reply:
Alright, but is there a way we can advertise that setting this value will lead to driver not looking at IMDS for calculating CSI Node allocatable, which is the bigger reasoning behind this option? 

Or would you also define that as an implementation detail. 

Trying to prevent the footgun or setting this parameter without realizing the underlying change in CSI Node allocatable calculation.